frontend/scripts/utils/getURLFromParams.js

```diff
@@ -70,6 +70,7 @@ function getURLForV3(endpoint, paramsArg = {}) {
   const queryParams = Object.keys(params).reduce((prev, current) => {
```

Comment:
why no use `query-string`

Reply:
sounds like a good idea, but maybe for another day